app/card_routes.py

```diff
@@ -0,0 +1,30 @@
+from app import db
+from flask import Blueprint, request, jsonify, make_response
+from app.models.card import Card
+from .route_helper import validate_model
+
+bp = Blueprint('cards', __name__, url_prefix="/cards")
+
+# update card endpoint
+@bp.route("/<card_id>", methods=["PATCH"])
```

Comment:
Since this isn't a general purpose PATCH endpoint, consider adding a trailing specifier similar to the mark (in)complete endpoints from task list.

```py
@bp.route("/<card_id>/like", methods=["PATCH"])
```